build.sbt

```diff
@@ -7,6 +7,18 @@ lazy val backend = (project in file("backend"))
   .dependsOn(core % "test->test;compile->compile")
   .settings(backendSettings:_*)
 
+lazy val localBackend = (project in file("local-backend"))
```

Comment:
@Horneth I think "yeah" to the second one but I'm not sure :) I meant to have some dir which holds the officially supported backends and then each supported backend has its own dir inside there